src/Pages/FieldInfoPage.js

```diff
@@ -236,10 +254,23 @@ function FieldInfoPage() {
       {/* custom table to show field info */}
       <TableComponent
         cols={columnData}
-        rows={getRowData()}
+        rows={fieldTableRows}
         deleteHandler={deleteField}
         editHandler={editField}
       />
+      <SimpleModal
+        isOpen={isEditModalOpen}
+        closeHandler={handleClose}
+        modalBody={
+          <FieldForm
+            operation='UPDATE'
+            title='Edit Field'
+            selectedRow={selectedRow}
+            isOpen={isEditModalOpen}
+            closeHandler={handleClose}
```

Comment:
Ideally, we should not need to pass these to both the SimpleModal and the inner component.
Let's see if we can improve on this.

Reply:
lets do this later